deploy/charts/cert-manager/templates/cainjector-deployment.yaml

```diff
@@ -65,6 +65,8 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          securityContext:
+            {{- toYaml .Values.cainjector.containerSecurityContext | nindent 12 }}
```

Comment:
If `.Values.cainjector.containerSecurityContext` is not set, what does this render as? We need to avoid rendering invalid YAML such as:

```
    ...
    securityContext:
```

(i.e. blocks containing no content)

Reply:
It's interesting. I thought this would work well since it's the default setup, but it renders as:

```
        securityContext:
          {}

```

I'll wrap them in an if statement.